scripts/incentive-spreadsheet-to-json.ts

```diff
@@ -127,15 +176,29 @@ async function convertToJson(
       `No low-income thresholds defined for ${state} - define them or turn off strict mode.`,
     );
   }
-  const response = await fetch(file.sheetUrl);
-  const csvContent = await response.text();
-  const rows = parse(csvContent, {
-    columns: true,
-    from_line: file.headerRowNumber ?? 1,
-  });
+
+  // any needed given output of csv-parse
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let rows: any;
```

Comment:
Couldn't it be `Record<string, string>[]`? That's the type of the `spreadsheetToJson` param that `rows` gets passed to.